test/framework/deployments/testserver/kubernetes.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/kumahq/kuma/test/framework"
 )
 
+const secondaryPort = 8080
```

Comment:
I've purposefully hardcoded this stuff as I wanted something simple, as it is used in single test, we can improve on this in the future if we need some more customizability